Library/Homebrew/extend/ENV/super.rb

```diff
@@ -164,7 +164,11 @@ def determine_isystem_paths
   end
 
   def determine_include_paths
-    keg_only_deps.map { |d| d.opt_include.to_s }.to_path_s
+    paths = keg_only_deps.map { |d| d.opt_include.to_s }
+    if compiler == :llvm_clang && Formula["llvm"].installed?
```

Comment:
Good question; that would be poor reasoning on my part, so that will be removed.

I guess in theory it depends on OpenMP being used, but I wasn't sure how to check `needs :openmp` and I figured since the directory only contains omp.h and libomp.dylib there wouldn't be any harm in including it for a formula that doesn't use OpenMP.